var/spack/repos/builtin/packages/r-pkgbuild/package.py

```diff
@@ -9,18 +9,19 @@
 class RPkgbuild(RPackage):
     """pkgbuild: Find Tools Needed to Build R Packages"""
 
-    homepage = "https://cran.r-project.org/package=pkgbuild"
-    url      = "https://cran.r-project.org/src/contrib/pkgbuild_1.0.3.tar.gz"
-    list_url = "https://cran.r-project.org/src/contrib/Archive/pkgbuild/"
+    homepage = "https://cloud.r-project.org/package=pkgbuild"
+    url      = "https://cloud.r-project.org/src/contrib/pkgbuild_1.0.3.tar.gz"
+    list_url = "https://cloud.r-project.org/src/contrib/Archive/pkgbuild/"
 
+    version('1.0.4', sha256='2934efa5ff9ccfe1636d360aedec36713f3bb3128a493241dbb728d842ea3b5f')
     version('1.0.3', sha256='c93aceb499886e42bcd61eb7fb59e47a76c9ba5ab5349a426736d46c8ce21f4d')
 
     depends_on('r@3.1:', type=('build', 'run'))
-    depends_on('r-callr', type=('build', 'run'))
+    depends_on('r-callr@2.0.0:', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('r-callr@3.2.0:', type=('build', 'run'))
```

Reply:
This changed to r-callr@3.2.0: with version 1.0.4, which was an update after I did the check. Added additional constraints.